Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.10.9-bullseye
+
+RUN apt-get update \
+    && apt-get upgrade -yq \
+    && apt-get install -yq --no-install-recommends \
+        nodejs \
+        npm
+
+ENV APP_DIR=/var/www/app
+WORKDIR ${APP_DIR}
+
+COPY requirements.txt ${APP_DIR}/requirements.txt
+
+RUN pip install pip-tools \
+    && pip-sync
+
+COPY package.json ${APP_DIR}/package.json
+
+RUN npm install \
+    && npm install -g gulp
+
+COPY rootfs /
+COPY . ${APP_DIR}
+
+RUN gulp local
+
+ENTRYPOINT ["/entrypoint.sh"]
+
+HEALTHCHECK \
```

Comment:
oh this is nice! didn't know about it

Reply:
You can run scripts for health checks as well @vanessa 

I've got one to check the available memory to kill off containers that run out of RAM.

```
HEALTHCHECK --interval=12s --timeout=12s --start-period=30s \
    CMD ./check-memory.sh
```

Where that script does
```
#!/bin/bash
musage=$(free | awk '/Mem/{printf("RAM Usage: %.2f%\n"), $3/$2*100}' |  awk '{print $3}' | cut -d"." -f1)
if [ $musage -ge 55 ]; then
echo "Current Memory Usage: $musage%"
exit 1
else
echo "Memory usage is in under threshold"
exit 0
fi
```